assets/misc/registries.yaml

```diff
@@ -1,13 +1,13 @@
 mirrors:
   registry.dso.mil:
     endpoint:
-      - "https://zarf.localhost"
```

Comment:
Seen this behavior in some restricted environments too, so just removing for now because it doesn't really do anything for us.  That, and it failed the E2E with the new mirroring setup: 
 https://repo1.dso.mil/platform-one/big-bang/apps/product-tools/zarf/-/jobs/7058776.